cmd/otel-allocator/internal/watcher/promOperator.go

```diff
@@ -89,7 +83,7 @@ func NewPrometheusCRWatcher(ctx context.Context, logger logr.Logger, cfg allocat
 				ProbeNamespaceSelector:          cfg.PrometheusCR.ProbeNamespaceSelector,
 				ServiceDiscoveryRole:            &serviceDiscoveryRole,
 				Version:                         "2.55.1", // fix Prometheus version 2 to avoid generating incompatible config
-				ScrapeProtocols:                 DefaultScrapeProtocols,
+				ScrapeProtocols:                 cfg.PrometheusCR.ScrapeProtocols,
```

Comment:
Alternative, instead of having ScrapeProtocols in PrometheusCR, we can use the ScrapeProtocols from the `PromConfig.GlobalConfig.ScrapeProtocols`, do you think that is cleaner?

Reply:
I am ok with that :)